src/experimental/build/commands/build/wasm_binary/rust/experimental_canister_template/Cargo.toml

```diff
@@ -8,10 +8,10 @@ crate-type = ["cdylib"]
 
 [dependencies]
 anyhow = "1.0.98"
-ic-cdk = "0.17.1"
-ic-cdk-timers = "0.11.0"
+ic-cdk = "0.18.1"
```

Comment:
should this be te beta version instead of the yanked version?

Reply:
I will push that now, it will actually be the alpha version of `0.18.2`

Reply:
This will have to be the last thing we do, DFINITY is working on some problems and there's a lot of yanking and incompatibility happening right now. I attempted but ran into issues. I say we just wait.

Reply:
We can just keep `0.18.1`, I have made this issue: https://github.com/demergent-labs/azle/issues/3046